net.xy.ykhwong.dosbox-daum.yml

```diff
@@ -0,0 +1,92 @@
+app-id: net.xy.ykhwong.dosbox-daum
```

Comment:
`ykhwong.xy.net` doesn't exist, and i doubt that `xy.net` is related to this

Reply:
The domain name is x-y.net but I can't have the - in the name there

Reply:
changed